api/config/swagger.py

```diff
@@ -378,7 +378,7 @@ def schema_view_swagger(request):
             'Questions': {
                 'Post question': coreapi.Link(
                     # fetch meetup id from the database
-                    url='/api/meetup/{meetup_id}/question/',
+                    url='/api/meetups/{meetup_id}/question/',
```

Comment:
Use the plural version of resources on the path, i.e. `/questions/` not `/question/`

Reply:
Thanks for that,
Let me fix it

Reply:
Fixed